assets/css/style.css

```
@@ -1321,48 +1021,39 @@ main {
   white-space: nowrap;
-  /* Prevent text wrapping */
-}
-
-.skills-item i,
-.skills-item ion-icon {
-  font-size: 20px;
-  color: var(--vibrant-green);
-  transition: var(--transition-1);
+  background: var(--surface-soft);
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)